data/varadbrid.json

```diff
@@ -0,0 +1,10 @@
+{
+    "title" :"Winter Soldier",
+    "image" :"https://images.app.goo.gl/5vLt1zHJfYPobtLLA",
+    "description" :"Childhood friend of Captain America who was experimented on by Hydra and then was provided a Metal Arm and Superhuman Abilities.",
+    "power" :["Metal Arm","Superhuman Abilities","Avenger"],
+    "widget_color" :"black",
+    "text_color" :"white", 
+    "dialogue":"How can I do something stupid ? You are taking all the stupid with you !!",

```

Comment:
Spacing non-uniformity found. spacing between ":" must be uniform throughout and must match with the general layout.